include/Unit.h

```diff
@@ -93,9 +93,9 @@ Unit* Unit_getCarrier(Unit* self);
 UnitIterator* Unit_getInterceptors(Unit* self);
 Unit* Unit_getHatchery(Unit* self);
 UnitIterator* Unit_getLarva(Unit* self);
-// TODO Unitset getUnitsInRadius(int radius, const UnitFilter &pred)
-// TODO Unitset getUnitsInWeaponRange(WeaponType weapon, const UnitFilter &pred)
-// TODO Unit* getClosestUnit(const UnitFilter &pred, int radius)
+UnitIterator* Unit_getUnitsInRadius(Unit* self, int radius, UnitFilterUnary pred);
+UnitIterator* Unit_getUnitsInWeaponRange(Unit* self, WeaponType weapon, UnitFilterUnary pred);
+Unit* Unit_getClosestUnit(Unit* self, UnitFilterUnary pred, int radius);
```

Comment:
Because I try to keen BWAPI names everywhere. Compliting predicate from pred when you see a filter type is not a great deal.